daemon/BUILD

```diff
@@ -4,6 +4,9 @@ cc_binary(
     deps = [
 	"//lib:ebpd"
     ],
+    data = [
```

Comment:
there's more to do then a simple data dependency, unfortunately. I've asked the bazel slack channel, but basically a plain data = ... will make the file available when a) you run tests, and b) when you use bazel run.

We need to link the data dependency in the file. I started looking into it today, we can sync up tomorrow. I'd leave it here for now (or remove, up to you), and add what's missing in a follow up change.

Reply:
I understand that "data = " brings in run time dependency and thought thats what you wanted. Are you talking about compile time dependency? Anyway we can discuss more.